File: weather/weatherJs/weather.js
function loadData(system) {

  // CURRENT CONDITIONS
  var $icon = $('#icon');
  var $temp = $('#temp');
  var $tempScale = $('#tempScale');
  var inputCity = system;

  //var accuweatherApiKey = "eOYiiAjNR0EuRaIGNoxAlXQQLn56cQMb";
  var accuweatherApiKey = "KrnJm3pGAtha40EFim82KLEqvaikzMeS"; // Accuweather api key2
  //var accuweatherApiKey = "XVmXE5NJJKoX17t4EucrFusDJSwYcStk"; // My api key
  var locationResourceURL = 'https://dataservice.accuweather.com/locations/v1/cities/search?apikey=' + accuweatherApiKey + '&q=' + inputCity;

  $.ajax({
    url: locationResourceURL,
    method: 'GET'
  }).done(function(result) { // Success
    
    
  var locationKey = result[0].Key; // Location key
  //var location = result[0].EnglishName; // City name

  var currentConditionsResourceURL = 'https://dataservice.accuweather.com/currentconditions/v1/' + locationKey + '?apikey=' + accuweatherApiKey + '&details=true';

      // Get current conditions (Accuweather Current Conditions API)
      $.ajax({
        url: currentConditionsResourceURL,
        method: 'GET'
      }).done(function(result) { // Success

        var temp;
        var tempScale;

        //Get the new icon image. All the image files are located in icons/conditions folder
        var icon = '../weather/icons/conditions/' + result[0].WeatherIcon + '.svg';
        

        //Get the new temperature
        temp = Math.round(result[0].Temperature.Imperial.Value).toString();

        //Get the Scale
        tempScale = '°F';

        //Change the temperature, scale and icon from the HTML Identifier.
        $temp.text(temp);
        $tempScale.text(tempScale);
        $icon.attr("src", icon);

      }).fail(function(err) { // Error handling
        console.log("error");
        throw err;
      });

    }).fail(function(err) { // Error handling
      console.log("error");
      throw err;
    });
  }

  //Always run this function with location name "Chico"
  loadData("Chico");

File: PyQtGUI/weather.py
import json
import time
import urllib.request
from config import YOUR_API

def get_weather():

    accuweatherApiKey = YOUR_API
    location_id = "Chico"
    locationKey = " "

    locationResourceURL = 'https://dataservice.accuweather.com/locations/v1/cities/search?apikey=' + accuweatherApiKey + '&q=' + location_id;
    data = " "

    with urllib.request.urlopen(locationResourceURL) as url:
        location_data = json.loads(url.read().decode())
        locationKey = location_data[0]["Key"]


    currentConditionsResourceURL = 'https://dataservice.accuweather.com/currentconditions/v1/' + locationKey + '?apikey=' + accuweatherApiKey + '&details=true';

    with urllib.request.urlopen(currentConditionsResourceURL) as url:
        weather_data = json.loads(url.read().decode())
        data = weather_data

    tempScale = '°F'
    return (data[0]['WeatherIcon'], round(data[0]['Temperature']['Imperial']['Value']), tempScale, location_id, location_data[0]['AdministrativeArea']['LocalizedName'])
